File: inventary.dart
void main() {
  Map<String, double> inventary = {
    'manzana': 1.5,
    'pan': 2.0,
    'leche': 3.5,
  }; //creo el objeto

  inventary['huevos'] = 4.0; //agrego un par clave valor
  print(inventary);
  inventary["pan"] = 2.5; //modifico un valor
  print(inventary);
  print('¿existe café?: ${inventary.containsKey("cafe")}'); //busco una clave
  inventary.remove('manzana'); //elimino un par clave valor
  print(inventary);
  double suma = inventary.values.reduce(
    (total, precio) => total + precio,
  ); //sumo todos los valores
  print('La suma es:  $suma');
}

File: inventary.py
"""Crud of dicctionary"""
inventary: dict = {  # creo el diccionario
    'manzana': 1.5,
    'pan': 2.0,
    'leche': 3.5
}
inventary['huevos'] = 4.0  # agrego una clave-valor

print(inventary)

inventary["pan"] = 2.5  # actualizo un valor
print(inventary)
print(f"¿existe café?: {"cafe" in inventary}")  # busco una clave
del inventary['manzana']  # elimino una clave
print(inventary)
# Aqui recorro todos los valores y sumo sin usar for
suma: float = sum(inventary.values())
print(suma)
